src/puppetlabs/puppetdb/query/report_data.clj

```diff
@@ -0,0 +1,31 @@
+(ns puppetlabs.puppetdb.query.report-data
+  (:require [puppetlabs.puppetdb.jdbc :as jdbc]
+            [puppetlabs.puppetdb.query-eng.engine :as qe]
+            [puppetlabs.puppetdb.scf.storage-utils :as sutils]
+            [puppetlabs.puppetdb.schema :refer [defn-validated]]
```

Comment:
Small nitpick: We've been favouring pulling in the `pls` namespace and using that for most cases of defn-validated, generally my opinion is that this makes lifting functions to other namespaces easier.